package.json

```diff
@@ -99,6 +99,7 @@
     "globby/fast-glob": "^3.3.2"
   },
   "dependencies": {
+    "@apidevtools/json-schema-ref-parser": "^11.9.1",
```

Comment:
I think we should move all of `src/platform/packages/shared/kbn-router-to-openapispec` to a `devOnly` package...

Then we can remove `joi-to-json` and avoid adding `@apidevtools/json-schema-ref-parser` if we wanna go ahead with this PoC.